src/components/projectStatus/ProjectTableContainer.tsx

```diff
@@ -148,13 +150,21 @@ const ProjectTableContainer: React.FC<Props> = props => {
           return newData;
         };
 
+        const tableGroup = props.tableGroupMap.get(project.tableGroupId);
+
         return (
           <div key={project.id}>
             <Title key={project.id} level={4}>
-              {props.isSponsor
-                ? `Project Name: ${project.name}`
-                : `${project.id} - ${project.name}`}
+              <a href={project.devpostUrl}>
+                {props.isSponsor
+                  ? `Project Name: ${project.name}`
+                  : `${project.id} - ${project.name}`}
+              </a>
```

Comment:
Let's make this an external link (opens in a new tab) so that it's easier to navigate back